docs/customresourcestate-metrics.md

```diff
@@ -193,6 +193,8 @@ spec:
             # whole objects may be copied into labels by prefixing with "*"
             # *anything will be copied into labels, with the highest sorted * strings first
             "*": [metadata, labels]
+            # a prefix before the asterisk will be used as a label prefix
```

Comment:
That sounds like a fair feature request to me. I'll have a look at the code later.

We also need to consider if this would add some complexity when converting to CEL